lang/german.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
```

Comment:
Since those files will not be launched by themselves we don't need shebang line at the top.